src/mappings/event/oracle.ts

```diff
@@ -40,13 +42,27 @@ export async function feedValues(
             updateValue: value,
         });
         let keyToString = key.__kind.toString();
+        let updateValueHuman : string = "";
         if (key.__kind === "ExchangeRate") {
             const exchangeCurrency = useLegacyCurrency
                 ? legacyCurrencyId.encode(key.value as CurrencyId_V15)
                 : currencyId.encode(key.value as CurrencyId_V17);
             update.typeKey = exchangeCurrency;
             keyToString += JSON.stringify(exchangeCurrency);
+            updateValueHuman = await convertAmountToHuman(update.typeKey, value);
         }
+        else { // FeeEstimation
+            if (process.env.SS58_CODEC === "kintsugi") {
+                updateValueHuman = await convertAmountToHuman( new NativeToken({ token: Token.KINT }), value);
+            }
+            else if (process.env.SS58_CODEC === "interlay") {
+                updateValueHuman = await convertAmountToHuman( new NativeToken({ token: Token.INTR }), value);
+            }
+            else {
+                console.error("Undefined SS58_CODEC");
```

Comment:
Please use
`ctx.log.error(...)`

Reply:
Also, probably a good idea to set updateValueHuman to some string, maybe empty. Otherwise saving to the database will fail since amountHuman has been defined with `{nullable: false}`

Reply:
Fixed these two things!